src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -0,0 +1,161 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.libr.Dao;
+import mate.academy.internetshop.model.Item;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ItemDaoJdbcImpl extends AbstractDao<Item> implements ItemDao {
+    private static Logger logger = Logger.getLogger(ItemDaoJdbcImpl.class);
+    private static String DB_TABLE_NAME = "internet_shop.items";
```

Comment:
```suggestion
    private static final String DB_TABLE_NAME = "internet_shop.items";
```
без final це не константа

Reply:
а я хотів спробувати написати константу DB_NAME = "internet_shop" в AbstractDao а потім в кожному JDBC impl додавати назву таблиці, так можна?

Reply:
@Levkosvd так, думаю це спрацює